blocks/library/freeform/format-list.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import clickOutside from 'react-click-outside';
+import classnames from 'classnames';
+import { fromPairs, omit } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, Dashicon } from 'components';
+
+/**
+ * Internal dependencies
+ */
+import './format-list.scss';
+
+function naiveCss2Jsx( styleText ) {
```

Comment:
Certainly can. I have added unit tests for the format stylings that TinyMCE is outputting.